src/Contact.js

```diff
@@ -0,0 +1,36 @@
+import React, {PropTypes} from 'react';
+import RemoveContactButton from './RemoveContactButton.js';
+import ContactName from './ContactName.js';
+
+const Contact = props => {
+  return (
+    <li className="contact">
+      <div className="image-cropper">
+        <img src={props.avatar} alt="avatar" />
+      </div>
+      <div className="contact-info">
+        <ContactName
```

Comment:
Great ideas, thank you! Yeah I had tried to set them as <a> elements but didn't understand how to pass the event objects through the chain to preventDefault(). I hadn't thought about stopPropagation() or using CSS to change the cursor to a pointer on hover, great ideas! Thanks for taking the time to test it out and give such great feedback!